README.md

```diff
@@ -11,44 +11,88 @@ MapStore has several example applications:
  * MapViewer is a simple viewer of preconfigured maps (optionally stored in a database using GeoStore)
  * MapPublisher has been developed to create, save and share in a simple and intuitive way maps and mashups created selecting contents by server like OpenStreetMap, Google Maps, MapQuest or specific servers provided by your organization or third party.
 
-For more information check the <a href="https://mapstore.readthedocs.io/en/latest/" target="_blank">MapStore documentation!</a>
+For more information check the [MapStore documentation!](https://mapstore.readthedocs.io/en/latest/)
 
 Download
 ------------
 You can download the WAR file from the latest release <a href="https://github.com/geosolutions-it/MapStore2/releases/latest" target="_blank">here</a>
 
-<a href="https://github.com/geosolutions-it/MapStore2/releases" target="_blank">All the releases</a>
+[All the releases](https://github.com/geosolutions-it/MapStore2/releases)
 
 Quick Start
 ------------
 
+
 There are two quick ways to test out MapStore. Either using <a href="https://www.docker.com/" target="_blank">Docker</a> (all tags are available in the [geosolutions dockerhub](https://hub.docker.com/r/geosolutionsit/mapstore2/tags?page=1&ordering=last_updated)) or a local java web container like <a href="http://tomcat.apache.org/" target="_blank">Apache Tomcat</a>
 
+
 ### Using Docker
+#### * Run Mapstore as standalone container 
+- Pull the latest image from Docker Hub:
 
-Pull the latest image from Docker Hub:
+  ``` shell
+  docker pull geosolutionsit/mapstore2
+  ```
 
-```
-docker pull geosolutionsit/mapstore2
-docker run --name mapstore -p8080:8080  geosolutionsit/mapstore2
-```
+- Run the docker command to create the container.
+  ```shell
+  docker run --name mapstore -p 8080:8080  geosolutionsit/mapstore2
+  ```
 
-Then you can access MapStore using the following URL:
+- Then you can access MapStore using the following URL:  
+  
+    [http://localhost:8080/mapstore](http://localhost:8080/mapstore)
 
-<a href="http://localhost:8080/mapstore" target="_blank">http://localhost:8080/mapstore</a>
 
+- Use the default credentials (**admin** / **admin**) to login and start creating your maps!
+
+#### * Run the Mapstore with postGIS through docker-compose in the local environment.
+
+---
+**Note**
+
+- To test a different release of MapStore, you should change the `MAPSTORE_WEBAPP_SRC` build argument in the docker-compose file.
+- You should change the value of `POSTGRES_PASSWORD` for more security. 
+- Due to proxy binding on host port 80, you may need to run docker-compose as root.
+
+---
+
+- To spin up the environment run:
+  ```shell
+  docker-compose up -d
+  ```
+
+---
+**Note**
+
+-  Take in consideration due to the requirements of the deployment, you should update the docker-compose to the latest version.
+---
+
+- After the docker-compose finish, you can access to the site using following URL:
+
+  [http://localhost:8080/mapstore](http://localhost:8080/mapstore)
```

Comment:
Fix documentation for docker-compose because it's exposed on standard HTTP port (80/tcp) and port 8080/tcp
```suggestion
  [http://localhost/mapstore](http://localhost/mapstore)
```